src/common/types.ts

```diff
@@ -171,6 +171,7 @@ export interface GameSettings {
   enableDXVKFpsLimit: boolean
   enableEsync: boolean
   enableFSR: boolean
+  enableMsync: boolean
```

Comment:
Whenever you're adding a new key to the settings, you *should* also add a default value for it (in `src/backend/config.ts`). See [this comment chain](<https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/3565#discussion_r1489992624>) for details on how this is done

It isn't strictly necessary to do this (we have enough type issues in the codebase right now that nothing yells at you for it), but it does add an "unknown" `undefined` into your key type ("unknown" because this is not actually reflected in the type)

Reply:
pushed a fix to this. Can you check again?

Reply:
that's not the code that I think CommandMC was talking about, look for enableFsync: isLinux in the source

Reply:
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/88222450/ff3eb412-685d-43c6-b733-6d232cac9156)
Well, in config.ts "isMac" is the default value already:

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/3630/files#diff-002cc40c00ed6f11f3ff2ccc3006de3bbda8a0aef5a18cc5fc29c2d1a55d59b7R316

Reply:
ok neat